WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -75,11 +75,15 @@ def users_search
     search(User)
   end
 
+  def regulations_search
+    search(Regulation)
```

Comment:
Yep it does work.
Isn't `serializable_hash` here to override Ruby's default json serialization, and provide a custom json description of the `ActiveRecord`?
It returns a hash, and here each regulation entry is already a hash from which we want all the fields to be serialized.
So in both cases we give to [render](https://github.com/cubing/worldcubeassociation.org/pull/722/files#diff-902940682572d690f1b6dc6195e6a233R63) an array of hashes, which seems consistent.

I honestly don't know how Rails internals work with jsonification, but it seems to deal with arrays and hashes automatically.